src/main/kotlin/io/emeraldpay/dshackle/data/BlockContainer.kt

```diff
@@ -91,12 +93,15 @@ class BlockContainer(
         if (timestamp != other.timestamp) return false
         if (full != other.full) return false
         if (transactions != other.transactions) return false
+        if (parentHash != other.parentHash) return false
```

Comment:
looks like parenthash sometimes can be emtpty (in case old vesion of dshckale at provider)